src/arch/LoadingWindow/LoadingWindow_GtkModule.cpp

```diff
@@ -31,7 +31,9 @@ extern "C" const char *Init( int *argc, char ***argv )
 	//gtk_window_set_icon( GTK_WINDOW(window), );
 	gtk_widget_realize(window);
 
-	splash = gtk_image_new_from_file(splash_image_path);
+	splash = gtk_image_new_from_pixbuf(
+		gdk_pixbuf_new_from_file (splash_image_path, nullptr)
+	);

```

Comment:
This was taken from https://github.com/stepmania/stepmania/pull/1854/commits/7ecc7b262b842a1cc1e66c02d597191c43a5b8fd. Perhaps this is caused by the fact, that my GTK3 was built against libpng 1.6.x.